src/vectorise.F90

```diff
@@ -1,23 +1,49 @@
 PROGRAM main
-    USE ISO_FORTRAN_ENV
+    ! NOTE: For documentation see:
+    !         * https://support.hdfgroup.org/documentation/hdf5/latest/group___f_h5.html
+    !         * https://support.hdfgroup.org/documentation/hdf5/latest/group___f_h5_f.html
+    !         * https://support.hdfgroup.org/documentation/hdf5/latest/group___f_h5_g.html
+    !         * https://support.hdfgroup.org/documentation/hdf5/latest/group___f_h5_l_t.html
+    !         * https://support.hdfgroup.org/documentation/hdf5/latest/group___f_h5_t.html
+    USE ISO_FORTRAN_ENV,    ONLY:   ERROR_UNIT,                                 &
```

Comment:
Now that this program is using many modules, lets be clear about where each external variable and routine is coming from.